src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -36,15 +37,23 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 			payload: payload,
 			log: logger,
 			action: payload.action,
-			...(!gitHubServerApp ? undefined : {
-				gitHubAppConfig: {
+			gitHubAppConfig: {
+				...(!gitHubServerApp ? {
+					gitHubAppId: undefined,
+					appId: parseInt(envVars.APP_ID),
+					clientId: envVars.GITHUB_CLIENT_ID,
+					gitHubBaseUrl: GITHUB_CLOUD_HOSTNAME,
+					gitHubApiUrl: GITHUB_CLOUD_API_BASEURL,
+					uuid: undefined
```

Comment:
i've not touched the GHE stuff so unsure, is gitHubAppId always a ghe server id?

can we just omit `gitHubAppId` and `uuid` instead of passing undefined?

Reply:
`Is gitHubAppId always a ghe server id?` <-- Yes, it is always the PK in the `GitHubServerApp` table.

`can we just omit gitHubAppId and uuid instead of passing undefined?` <-- I suppose so, from code point of view, I think they are the same? Not sure what's the team's convention for `undefined`?

Reply:
Just talked to GHE team, we don't have an convention. so passing in `undefined` as well to maintain a similar structure.